activitypub.php

```diff
@@ -31,74 +31,66 @@ function init() {
 
 	\define( 'ACTIVITYPUB_OBJECT', 'ACTIVITYPUB_OBJECT' );
 
-	require_once \dirname( __FILE__ ) . '/includes/table/class-followers.php';
-	require_once \dirname( __FILE__ ) . '/includes/class-http.php';
-	require_once \dirname( __FILE__ ) . '/includes/class-signature.php';
-	require_once \dirname( __FILE__ ) . '/includes/class-webfinger.php';
-	require_once \dirname( __FILE__ ) . '/includes/peer/class-followers.php';
-	require_once \dirname( __FILE__ ) . '/includes/functions.php';
-
-	require_once \dirname( __FILE__ ) . '/includes/model/class-activity.php';
-	require_once \dirname( __FILE__ ) . '/includes/model/class-post.php';
-	require_once \dirname( __FILE__ ) . '/includes/model/class-follower.php';
-
-	require_once \dirname( __FILE__ ) . '/includes/class-migration.php';
 	Migration::init();
-
-	require_once \dirname( __FILE__ ) . '/includes/class-activity-dispatcher.php';
 	Activity_Dispatcher::init();
-
-	require_once \dirname( __FILE__ ) . '/includes/class-activitypub.php';
 	Activitypub::init();
-
-	require_once \dirname( __FILE__ ) . '/includes/collection/class-followers.php';
 	Collection\Followers::init();
 
 	// Configure the REST API route
-	require_once \dirname( __FILE__ ) . '/includes/rest/class-outbox.php';
 	Rest\Outbox::init();
-
-	require_once \dirname( __FILE__ ) . '/includes/rest/class-inbox.php';
 	Rest\Inbox::init();
-
-	require_once \dirname( __FILE__ ) . '/includes/rest/class-followers.php';
 	Rest\Followers::init();
-
-	require_once \dirname( __FILE__ ) . '/includes/rest/class-following.php';
 	Rest\Following::init();
-
-	require_once \dirname( __FILE__ ) . '/includes/rest/class-webfinger.php';
 	Rest\Webfinger::init();
 
-	// load NodeInfo endpoints only if blog is public
-	if ( true === (bool) \get_option( 'blog_public', 1 ) ) {
-		require_once \dirname( __FILE__ ) . '/includes/rest/class-nodeinfo.php';
-		Rest\NodeInfo::init();
-	}
-
-	require_once \dirname( __FILE__ ) . '/includes/class-admin.php';
 	Admin::init();
-
-	require_once \dirname( __FILE__ ) . '/includes/class-hashtag.php';
 	Hashtag::init();
-
-	require_once \dirname( __FILE__ ) . '/includes/class-shortcodes.php';
 	Shortcodes::init();
-
-	require_once \dirname( __FILE__ ) . '/includes/class-mention.php';
 	Mention::init();
-
-	require_once \dirname( __FILE__ ) . '/includes/class-debug.php';
 	Debug::init();
-
-	require_once \dirname( __FILE__ ) . '/includes/class-health-check.php';
 	Health_Check::init();
+	Scheduler::init();
+}
+\add_action( 'plugins_loaded', '\Activitypub\init' );
+
+/**
+ * Class Autoloader
+ */
+spl_autoload_register(
+	function ( $class ) {
+		$base_dir = trailingslashit( __DIR__ ) . 'includes' . DIRECTORY_SEPARATOR;
```

Comment:
Just catches my eye that you are using `trailingslashit` which adds slash but then `DIRECTORY_SEPARATOR` which could be a backslash on Windows.